User-Help/FormattingHelp.md

```diff
@@ -0,0 +1,104 @@
+
+```
+
+  You can check out [Codidact][1] for an open-source, community-oriented Q&A platform!
+  
+  [1]: https://codidact.org`
+
+```
+  Which will render like this:
+
+  > You can check out [Codidact][1] for an open-source, community-oriented Q&A platform!
+  > 
+  > [1]: https://codidact.org
+
+  If you'd like to include rollover text for your link, you can put your text, surrounded by quote marks (`"`), into the URL of your link. For instance, `[Codidact](https://codidact.org "The open-source Q&A platform")` will create a link with `The open-source Q&A platform` as the rollover text.
+
+* **Blockquotes**
+
+  When quoting text from somewhere else, you can use a blockquote to signify that this is not your original text, but a quote from somewhere else. Put a greater than symbol (`>`) before a paragraph to mark that quote as a blockquote. For instance, `> This is a blockquote.` will render as a quote:
+
+  > This is a blockquote!
+
+  If you want to include a second paragraph as part of the same quote, you have to include another `>` before each line, like this:
+
+```
+  > This is a blockquote!
+  >
+  >This is part of the same quote!
+```
+
+  This will render as a single blockquote:
+
+  > This is a blockquote!
+  >
+  > This is part of the same quote!
+
+  See also the [citation and referencing help](/help/referencing) for guidelines on how to properly reference different quotes from various places.
+
+* **Images**
+
+  Embedding images in a post is very similar to the formatting for links (detailed above). The format `![alt text](image URL)` will embed an image in your post. Please remember to write alt text for your image! Codidact allows you to upload an image directly, or you can embed an external image from image hosting sites such as Imgur.
+
+* **Lists**
+
+  You can generate either an ordered list or a bullet list. For a bullet list, you can use either `*` or `-` to create your list.
+
+```
+* List item one
+* List item two
+```
+
+  This will render like this:
+
+>  * List item one
+>  * List item two
+
+  If you want to create a numbered list, use `1.`, `2.`, etc instead of `*` or `-`.
+
+* **Horizontal line**
+
+  You can use a horizontal line for separating sectiond of your post. Use `---` on its own line to generate a horizontal line.
+
+* **Headings**
+
+  You can include a heading in your post. Using a single hash symbol (`#`) before a line of your post will create a large heading, while using two will create a slightly smaller heading.
+
+```
+# Heading 1
+
+## Heading 2
+```
+
+  > # Heading 1
+  >
+  > ## Heading 2
+
+* **Subscript and superscript**
+
+  While subscript and superscript isn't directly supported in Markdown, you can use the HTML tags `<sub>` and `<sup>`. `<sup>1</sup>` will create superscript, which can be used to create footnotes<sup>1</sup>.
```

Comment:
We have first-class support for footnotes, so superscript should probably be discouraged for it. Footnote syntax:

```
here are some words[^1]

[^1]: a footnote
```